apis/clusterresources/v1beta1/clusterbackup_types.go

```diff
@@ -71,6 +72,14 @@ func (cbs *ClusterBackupStatus) UpdateStatus(instBackup *models.BackupEvent) {
 	cbs.Progress = fmt.Sprintf("%f", instBackup.Progress)
 }
 
+func (cb *ClusterBackup) AttachToCluster(id string) {
+	cb.Status.ClusterID = id
+}
+
+func (cb *ClusterBackup) DetachFromCluster() {
+
+}
+
```

Comment:
Delete if unused, please

Reply:
It was created to implement the custom interface `Object` which allows to work generic object status